labs-ios-starter/CityNetworkClient.swift

```diff
@@ -50,4 +50,63 @@ class CityNetworkClient {
             completion(.success(true))
         }.resume()
     }
+    
+//    // fetch rent
+//    {
+//      "msg": "average rent",
+//      "score": 4,
+//      "avg_rent": 2100
+//    }
+    func fetchAverageRent(forCity city: String, completion: @escaping (Result<Bool, NetworkError>) -> Void) {
+        
+        let averageRentForCityURL = baseDSURL.appendingPathComponent("city_scr/\(city)")
+        
+        URLSession.shared.dataTask(with: averageRentForCityURL) { (data, _, error) in
+            if let error = error {
+                print("Error \(error)")
+            }
+            
+            guard let data = data else {
+                print("No Data Returned from API")
```

Comment:
Add `.completion(.failure(.noData))` after the `print`.

It looks like these same changes need to be made on the above function `fetchAllCities()` as well.